templates/default/src/components/organisms/PageTitle/PageTitle.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Head from 'next/head'
+
+const PageTitle = ({ title }) => {
```

Comment:
change to title since that is the one thing that we may want to change from components outside document.js

Reply:
Is the key really necessary? I don't think eslint-react nor react is going to issue a warning here since it's wrapped in the <Head> element